aspnet/includes/signalr/signalr-version-disambiguation.md

```diff
@@ -1,2 +1,2 @@
-> [!NOTE]
-> This article refers to ASP.NET SignalR. If you're thinking about using SignalR to enable real-time scenarios with Java, Node.js, or in a serverless scenario, take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction). If you've already used ASP.NET SignalR, take a look at the [version differences](/aspnet/core/signalr/version-differences) page to understand the differences in the versions and the improvements in ASP.NET Core SignalR. Finally, if you know you'll be running your real-time apps in Microsoft Azure, take a look at the [Azure SignalR Service](/azure/azure-signalr/signalr-overview), as it provides cloud-based scale-out once your apps need it.
\ No newline at end of file
+> [!WARNING]
+> This documentation is not for the latest version of SignalR. Take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction) to keep up with the latest updates. 
```

Comment:
"latest updates" makes it sound like Asp.Net Core SignalR is an update of Asp.Net SignalR